contracts/packages/unwind-permit/UnwindPermit.sol

```diff
@@ -15,10 +30,25 @@ contract UnwindPermit is EIP712 {
     bytes32 private constant _PERMIT_TYPEHASH =
         keccak256("UnwindPermit(address owner,uint256 orderID,uint256 value,uint256 nonce,uint256 deadline)");
 
+    // Unwind permit keeper
+    address public keeper;
+
+    // Current OTC wrapper address that is allowed to call checkOrderPermit()
+    address public currentOTCWrapper;
+
     /**
      * @dev Initializes the {EIP712} domain separator using the `name` parameter, and setting `version` to `"1"`.
      */
-    constructor(string memory name) EIP712(name, "1") {}
+    constructor(string memory name, address _keeper) EIP712(name, "1") {
```

Comment:
can pass in otc wrapper in constructor as well

Reply:
We are using UnwindPermit.sol on the constructor of OTCWrapperV2.sol as an immutable - hence that here we use setOTCWrapper